profile_library/signify/LCL002/model.json

```diff
@@ -0,0 +1,15 @@
+{
+    "calculation_strategy": "lut",
+    "created_at": "2024-06-20T11:25:39.819433",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plus Plug S",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.12.6:docker"
+    },
+    "name": "Hue lightstrip outdoor",
+    "standby_power": 0.25
+    "config_flow_discovery_remarks": "This was tested with included PSU. If you run this with a PSU that is different from the inlcuded PSU this will skew the readings."
```

Comment:
```suggestion
    "config_flow_discovery_remarks": "This profile was measured with the default included PSU. If you run this with a PSU that is different from the included PSU this can skew the readings."
```